### ISSUE TITLE
Widget options do not accept dot `.`

### ISSUE THREAD

**Issue by bropines** [dmMaze/BallonsTranslator#697]
If you try, for example, to enter a dot in the GPT translator...

Like fractional values ​​`0.1` do not fit

**Comment by dmMaze** [comment]
Which parameter are you referring? I just found top p can be float and fixed it 15b3f9fb2a6b002d545a96b69ae827be539aae65

**Comment by bropines** [comment]
Yes, that's it. Basically, I figured out how to fix it in my modules if it appears.
